7:18:56.102938+00:00 app[web.1]:     func(event)
2022-05-11T17:18:56.102939+00:00 app[web.1]:   File "/app/app.py", line 47, in handle_message
2022-05-11T17:18:56.102939+00:00 app[web.1]:     print(f.read())
2022-05-11T17:18:56.102940+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.10/codecs.py", line 322, in decode
2022-05-11T17:18:56.102940+00:00 app[web.1]:     (result, consumed) = self._buffer_decode(data, self.errors, final)
2022-05-11T17:18:56.102940+00:00 app[web.1]: UnicodeDecodeError: 'utf-8' codec can't decode byte 0xc0 in position 28: invalid start byte
2022-05-11T17:18:56.103329+00:00 app[web.1]: 10.1.35.12 - - [11/May/2022 17:18:56] "[35m[1mPOST /callback HTTP/1.1[0m" 500